src/main/java/ch/wisv/events/ChConnectConfiguration.java

```diff
@@ -69,7 +70,9 @@ public class ChConnectConfiguration {
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http
                 .cors(Customizer.withDefaults())
-                .csrf(Customizer.withDefaults())
+                .csrf((csrf) -> csrf
+                        .csrfTokenRequestHandler(new XorCsrfTokenRequestAttributeHandler())
+                )
```

Comment:
I would merge these changes with the `csrf -> csrf...` on line 81 as that is the reason they use a builder design pattern for setting this config.

Also the .csrf() below this one could be the case why disabling it before did not work (as it was enabled again a couple lines below) so merging should also possibly prevent these mistakes from happening in the future.